bin/tex2stree

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env node
+
+/*************************************************************************
+ *
+ *  tex2mml
```

Comment:
You should probably change this to tex2stree, and adjust the comment below that describes what it is.